vllm/model_executor/models/llama.py

```diff
@@ -48,6 +48,7 @@
 from vllm.model_executor.sampling_metadata import SamplingMetadata
 from vllm.sequence import SamplerOutput
 from vllm.utils import is_hip, print_warning_once
+from vllm._C import ops
```

Comment:
As before, handle the Pybind methods inside `_custom_ops.py`:
https://github.com/ROCm/vllm/blob/fp8-gemm/vllm/_custom_ops.py

Reply:
ok

Reply:
But I still think we should get rid of this specific weight loading function for quantized weight.

Reply:
Oh, ops is not used in llama.py, can be removed.